src/pykeen/nn/node_piece/representations.py

```diff
@@ -298,3 +315,52 @@ def _plain_forward(
             ),
             self.aggregation_index,
         )
+
+    def ratio_unique_hashes(self) -> RatioInfo:
```

Comment:
also maybe we can rename this function to describe what it's for rather than how it's implemented, like `estimate_diversity`

Reply:
Done in 7afa1b8b